requirements.txt

```diff
@@ -1,13 +1,15 @@
 matplotlib>=3.0.0
 pandas>=0.23.4
 msgpack-numpy>=0.4.3.2
-spacy>=2.3.5
+spacy>=3.0
 scipy>=1.1.0
 scikit-learn>=0.20.0
 nltk>=3.4
 dill>=0.2.9
 torch>=1.2.0
 clean-text>=0.1.1
 joblib>=0.13.2
+pymongo>=4.0
 tqdm
-
+pyyaml>=5.4.1
+dnspython>=1.16.0
```

Comment:
Technically, packages that have a `setup.py` don't need to have a `requirements.txt` -- that's what `install_requires` in `setup.py` is for. 

I can make this change after 3.0. But `setup.py` needs to be updated to include these new packages.